hal/inc/pinmap_hal.h

```diff
@@ -99,8 +99,6 @@ STM32_Pin_Info* HAL_Pin_Map(void);
 
 // todo - this is corev1 specific, needs to go in a conditional define
 
-#define LED1 LED_USER
```

Comment:
Probably not a best fix possible, but linker started to generate warning (or error in our case) due to `LED1` define breaking ODR during LTO. `LED1` value is already defined as part of `Led_TypeDef` enum.